### PR TITLE
cAPI driven logos in commercial containers fix

### PR DESCRIPTION
**Pull request by Calanthe**
## What does this change?

This PR fixes the issue with badges shown incorrectly in single and multi branded containers.

I had to change markup in the `itemCard.scala.html` file. I want to have two separate links: one for item and another for badge. I hope that his is ok, I didn't find any bugs on the `/commercial-containers` page. But it still requires proper testing.

Changes in the `itemLargeCard.scala.html` and `itemSmallCard.scala.html` will be done later.
## Screenshots

Before:
Single branded component:
upload_23_8_2016_at_09_03_54

Multi branded component:
upload_23_8_2016_at_08_50_48

After:
Single branded component:
screen shot 2016-09-06 at 16 24 21

Multi branded component:
screen shot 2016-09-06 at 16 24 32
## Request for comment

@regiskuckaertz @kelvin-chappell

### CODE REVIEW

File: static/src/stylesheets/module/commercial/_adverts.scss [hunk 1]
```diff
@@ -737,10 +737,10 @@
         float: none;
         min-height: 0;
         overflow: visible;
-        margin: auto $gs-gutter / 4 0;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_adverts.scss]: This is weird. Why is the margin moving from left-right to bottom?

File: static/src/stylesheets/module/commercial/_adverts.scss [hunk 2]
```diff
@@ -737,10 +737,10 @@
...
+        margin: auto 0 $gs-gutter / 4 0;
         padding: 0;
         position: static;
-        width: auto;
+        width: 96%;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_adverts.scss]: What is that? 😃

File: common/app/views/fragments/commercial/contentLogo.scala.html [hunk 3]
```diff
@@ -1,11 +1,10 @@
 @(branding: common.commercial.Branding)
 
-<div class="brandbadge brandbadge--onside ad-slot--force-display ad-slot--paid-for-badge--interactive">
-    <div class="brandbadge__inner">
+<div class="brandbadge brandbadge--onside ad-slot--force-display ad-slot--paid-for-badge ad-slot--paid-for-badge--front">
```
Comment (regiskuckaertz) [inline common/app/views/fragments/commercial/contentLogo.scala.html]: Can we use [this markup](https://github.com/guardian/frontend/blob/master/common/app/views/commercial/containerWrapper.scala.html#L29) here? Much simpler and also does not use `H3` which does not make sense.

Comment (regiskuckaertz) [inline common/app/views/fragments/commercial/contentLogo.scala.html]: And [here](https://github.com/guardian/frontend/blob/3be9765e2b52c887909ccb0b4dc11b6101c6ec30/commercial/app/views/contentapi/items.scala.html#L85)

**Comment by kelvin-chappell** [comment]
Looks good to me.  Thank you!
Getting all the card sizes working is going to be much easier once we get some data coming through in prod capi, but this will be enough for launch.
I'll let Regis give the thumbs up when he's ready.

File: static/src/stylesheets/module/commercial/_adverts.scss (hunk 1 above)
Comment (Calanthe) [inline static/src/stylesheets/module/commercial/_adverts.scss]: I changed it a little now. I want the logo to be in the right bottom corner. The width is 100% and I added extra padding-right in the div inside.

File: common/app/views/fragments/commercial/contentLogo.scala.html (hunk 3 above)
Comment (Calanthe) [inline common/app/views/fragments/commercial/contentLogo.scala.html]: Good points. I changed the markup temporary just to check how works and it broke badges everywhere ;/. As I don't have time for refactoring styles and test it everywhere on the page, I will make those changes later in another PR.

Comment (regiskuckaertz) [inline common/app/views/fragments/commercial/contentLogo.scala.html]: Ok, then.

**Comment by regiskuckaertz** [comment]
👍

**Pull request merged**